rules/windows/privilege_escalation_exploit_cve_202238028.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2024/04/23"
+integration = ["endpoint", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/04/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a privilege escalation attempt via exploiting CVE-2022-38028 to hijack the print spooler service execution.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.file-*", "logs-windows.sysmon_operational-*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential privilege escalation via CVE-2022-38028"
+references = [
+  "https://www.microsoft.com/en-us/security/blog/2024/04/22/analyzing-forest-blizzards-custom-post-compromise-tool-for-exploiting-cve-2022-38028-to-obtain-credentials/"
+]
+risk_score = 73
+rule_id = "dffbd37c-d4c5-46f8-9181-5afdd9172b4c"
+severity = "high"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Privilege Escalation", "Tactic: Defense Evasion", "Data Source: Elastic Endgame", "Data Source: Elastic Defend", "Data Source: Sysmon"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where host.os.type == "windows" and
```

Comment:
Any way to make this slightly more specific through the addition of event actions and/or non-system users? But nice!

Reply:
both deletion and creation are useful IoCs